README.md

```diff
@@ -78,6 +92,37 @@ For information about contributing to this project please refer to this
 * Releasing new versions
 * Docker development
 
+## CI/Testing Strategy
+
+The following parameters are considered in the testing strategy
+in the CI for this project:
+### Database
+
+We are testing this starter kit against PostgreSQL and MySQL.
+### Ruby Version
+
+We are testing this starter kit against the latest Ruby version
+and the last minor of the previous major. Eg: 3.2 and 2.7.
```

Comment:
Shouldn't we test against ruby versions that are still maintained, regardless of the major/minor number? For instance, once v3.3 is out, and v2.7 is EOL, we should test for v3.0, v3.1 & v3.2. That'd also be aligned with our new policy for main Solidus.

Reply:
Yes, you are right!